oval_graph/xml_parser.py

```diff
@@ -24,10 +25,14 @@ def __init__(self, src):
         self.root = self.tree.getroot()
         if not self.validate(
                 'schemas/arf/1.1/asset-reporting-format_1.1.0.xsd'):
+            sys.stdout.write("\033[1;31m")
```

Comment:
You open this red section, but you also have to close it, otherwise everythig that follows turns red.